design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -0,0 +1,38 @@
+type ComparableItem {
+    productId: ID!
+    name: String!
+    priceRange: ProductPriceRange!
+    url: ProductUrl
+    images: [ProductImage]
+    values: [ProductAttribute]
+}
+
+type ComparableIndicator {
+    code: String!
+    title: String!
+}
+
+type CompareList {
```

Comment:
Please provide usage examples (preferably with screenshots of the corresponding pages).

Reply:
added scenario